src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -63,7 +63,7 @@ public function isProxyCandidate(Definition $definition)
     /**
      * {@inheritdoc}
      */
-    public function getProxyFactoryCode(Definition $definition, $id)
+    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null)
```

Comment:
well... you could do play with `func_get_args` and `func_num_args` : https://github.com/symfony/validator/blob/2.8/Validator/RecursiveValidator.php#L118 it isn't pretty though...